code/datums/spells/construct_spells.dm

```diff
@@ -64,7 +64,11 @@
 	action_icon_state = "summonsoulstone"
 	action_background_icon_state = "bg_demon"
 
+	if(iscultist(user) || iswizard(user))

```

Comment:
i'm guessing this wont compile because user is never actually defined anywhere in this file